sdk/python/foundation-models/system/inference/automatic-speech-recognition/asr-batch-endpoint.ipynb

```diff
@@ -162,7 +164,7 @@
    "outputs": [],
    "source": [
     "model_name = \"openai-whisper-large\"\n",
-    "model_version = \"1\"\n",
+    "model_version = \"4\"\n",
```

Comment:
It might be better to get the latest version of the model in the registry instead of hardcoding it.

Reply:
I'll merge with hard-coded first. If @ManojBableshwar  wants to use latest in a future update, he can do that.